source/common/http/utility.h

```diff
@@ -372,6 +372,15 @@ bool sanitizeConnectionHeader(Http::RequestHeaderMap& headers);
  */
 const std::string& getProtocolString(const Protocol p);
 
+/**
+ * Constructs the original URI sent from the client from
+ * the request headers.
```

Comment:
add comment on 
it is consists of :
   ForwardedProto + Host + EnvoyOriginPath

Reply:
Done